### DISCUSSION TITLE
Haven't Been Approved?

### DISCUSSION THREAD

**Discussion by laulpogan** [community/community#13280] [Other Feature Feedback, Questions, & Ideas]
Hi,

I applied for sponsors over a week ago from the US and still haven't been approved? Is there support or someone I can talk to about this?

**Comment by jlord** [comment]
Hiya! Looks like you're set up now?

**Comment by laulpogan** [comment]
Yup! Thank you! Currently trying to see if I can hack together a docx viewer on the site. You’d think it be a given considering 😅


Fond Regards,
Paul
